src/Server.ts

```diff
@@ -87,7 +88,8 @@ class Server {
     const api = APIService.getInstance();
 
     if (api.config.env === "development") {
-      app.get("/docs", DocsHandler);
+      app.get("/api_docs", DocsHandler);
```

Comment:
```suggestion
      app.get("/api-docs", DocsHandler);
```
I think using hyphens instead of underscores would be more REST. 

  Maybe it will be `/api-info`, `/api-metadata`, or simply `/metadata` because it can be difficult for the developer to understand the difference between `/docs` and `/api-docs` as they are already working on the API

Reply:
That's a great idea. `/metadata` would be great. I will fix it.